x-pack/test/reporting_api_integration/reporting_and_security/security_roles_privileges.ts

```diff
@@ -165,23 +164,21 @@ export default function ({ getService }: FtrProviderContext) {
     describe('Discover: Generate CSV report', () => {
       it('does not allow user that does not have the role-based privilege', async () => {
         const res = await reportingAPI.generateCsv(
```

Comment:
Decided to move the `username` and `password` parameters to the end of the arguments. This will be useful for more `csv_searchsource` testing in later PRs.